api/swig/python/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Why are we setting this only here? Setting it at a higher CMake would reduce verbosity also in C++ setup: https://cmake.org/cmake/help/latest/variable/CMAKE_CXX_STANDARD.html

Reply:
Agreed. I'll refactor this into the root CMakeLists.txt

Reply:
Resolved: https://github.com/OpenNavigationSurface/BAG/commit/049f43a6004464aa6e5c2acf04eac7b55d46a8a1